__tests__/errorReporting.spec.js

```diff
@@ -396,6 +396,46 @@ describe('error reporting', () => {
         });
     });
 
+    it('should make a reporting network request after queuing a report and handle an empty response', () => {
+      expect.assertions(4);
+      const testError = {
+        message: 'test error',
+        stack: '1\n2\n3',
+      };
+      const otherData = { a: 1, b: 2 };
+
+      const store = createStore(
+        combineReducers({
+          config: () => fromJS({ reportingUrl: '/home' }),
+          errorReporting: reducer,
+        }),
+        applyMiddleware(thunk.withExtraArgument({ fetchClient: fetch }))
+      );
+
+      fetch.mockResponseOnce();
+
+      const expectedReports = [
+        {
+          msg: testError.message,
+          stack: testError.stack,
+          href: 'about:blank',
+          otherData,
+        },
+      ];
+
+      return store.dispatch(addErrorToReport(testError, otherData))
+        .then(() => {
+          expect(fetch).not.toThrow();
```

Comment:
Wondering if this line would yield the expected behavior. Normally a callback `expect(() => fetch()).not.toThrow()` would be a safer bet. Also, being that `fetch` is asynchronous, are we capturing the error if it were to throw (or reject) for fetch?